lib/src/main/java/dev/doglog/DogLog.java

```diff
@@ -28,10 +27,10 @@
 
 /** A logger based on WPILib's {@link DataLogManager} */
 public class DogLog {
+  public static final String USAGE_REPORT = "DogLog";
```

Comment:
why make this public?

Reply:
because this in particular was copied directly from choreo's port without further consideration. I'll make it private.